docs/book/user-guide/advanced-guide/configure-steps-pipelines.md

```diff
@@ -387,20 +389,20 @@ run_name: my_run
 schedule: { }
 settings: { }  # same as pipeline settings
 steps:
-  name_of_step_1:
+  step_invocation_id:
     settings: { }  # same as step settings
-  name_of_step_2:
+  other_step_invocation_id:
     settings: { }
   ...
 ```
 
-ZenML provides a convenient method that takes a pipeline instance and generates a config template based on its settings automatically:
-
+You can also use the following method to generate a config template for your pipelines:
```

Comment:
Could be a bit clearer that the resulting template is custom-tailored to this pipeline

Reply:
Also, does the template already include the currently configured options as defaults somehow? (not sure if it should, but if it does then maybe mention that)

Reply:
Good point, I'll mention that this is for the specific pipeline/stack combination!

Regarding the second point: No, but the existing configuration in code (which would also exist the next time the user uses that pipeline) will be merged with the config file.